pallets/asset-manager/src/lib.rs

```diff
@@ -401,13 +502,13 @@ pub mod pallet {
         #[transactional]
         pub fn set_units_per_second(
             origin: OriginFor<T>,
-            #[pallet::compact] asset_id: AssetId,
+            asset_id: T::AssetId,
```

Comment:
why not compac? it's still just a `u128`

Reply:
Yeah not sure why it got removed, but yeah it's still u128 so can be added to all the AssetId extrinsic params.